aztk/cli.py

```diff
@@ -32,14 +31,14 @@ def main():
     try:
         run_software(args)
     except batch_error.BatchErrorException as e:
-        util.print_batch_exception(e)
-    except error.AztkError as e:
+        utils.print_batch_exception(e)
+    except aztk_sdk.utils.error.AztkError as e:
```

Comment:
any reason error modules is under utils?

Reply:
not really -- right now there is just models and client in the root, with everything else in utils.